tests/swaggervalidate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set +e
+
+SWAGGERFILE=https://raw.githubusercontent.com/vmware/harbor/$1/docs/swagger.yaml
+VALIDATOR=http://online.swagger.io/validator/debug?url=$SWAGGERFILE
+
+echo $SWAGGERFILE
+
+TIMEOUT=10
+while [ $TIMEOUT -gt 0 ]; do
+    STATUS=$(curl --insecure -s -o /dev/null -w '%{http_code}' $VALIDATOR)
+    if [ $STATUS -eq 200 ]; then
```

Comment:
curl will return non-zero if the connection is time-out.
if the status code should be 200 only, no need to check its value.
Or there maybe other non-200 status code? that indicate other errors?